.gitignore

```diff
@@ -36,3 +36,4 @@ cucumber-pro.log
 
 # Temp files of yar
 .yardoc
+docker.gemrc
```

Comment:
This doesn't make sense. I would like to be able to build the image locally. What I suggest:
Create a directory `.docker/latest` (latest -> docker tag) and place the file there. This is practise I normally use to have files for each tag available.